circle.yml

```diff
@@ -14,3 +14,14 @@ jobs:
       - run:
           name: Compile the contract
           command: make
+  test:
+    docker:
+      - image: nixorg/nix:circleci
```

Comment:
Is this automatically downloading `dapp`?

Reply:
no, this is getting a docker image with `nix` installed. `dapp` is installed when entering the `nix-shell`